src/main/scala/mesosphere/marathon/MarathonScheduler.scala

```diff
@@ -144,9 +148,19 @@ class MarathonScheduler @Inject() (
     }
   }
 
+  protected def updateDefaultRegion(masterInfo: MasterInfo): Unit = {
```

Comment:
It is called home region in the design doc so maybe we should stick with that? https://github.com/mesosphere/marathon-design/commit/813c6a1f303b12edb65a0ca1c8eae41c8f6e9d7f#diff-af3ac823b06139f90539055737e0a188

Reply:
Yeah, let's stick with home region nomenclature; I think this is what they are calling it in other DCOS components